2018-08-16 21:51:05,331 DEBUG    root            Loaded Command Group: [u'gcloud', u'auth']
2018-08-16 21:51:05,332 DEBUG    root            Loaded Command Group: [u'gcloud', u'auth', u'login']
2018-08-16 21:51:05,334 DEBUG    root            Running [gcloud.auth.login] with arguments: []
2018-08-16 21:51:05,336 INFO     ___FILE_ONLY___ Go to the following link in your browser:

    https://accounts.google.com/o/oauth2/auth?redirect_uri=urn%3Aietf%3Awg%3Aoauth%3A2.0%3Aoob&prompt=select_account&response_type=code&client_id=32555940559.apps.googleusercontent.com&scope=https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fuserinfo.email+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fappengine.admin+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcompute+https%3A%2F%2Fwww.googleapis.com%2Fauth%2Faccounts.reauth&access_type=offline



2018-08-16 21:51:07,184 INFO     oauth2client.client Failed to retrieve access token: {
  "error": "invalid_request",
  "error_description": "Missing required parameter: code"
}
2018-08-16 21:51:07,185 ERROR    root            There was a problem with web authentication.
2018-08-16 21:51:07,186 DEBUG    root            (gcloud.auth.login) invalid_requestMissing required parameter: code
Traceback (most recent call last):
  File "/tmp/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 848, in Execute
    resources = calliope_command.Run(cli=self, args=args)
  File "/tmp/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 770, in Run
    resources = command_instance.Run(args)
  File "/tmp/google-cloud-sdk/lib/surface/auth/login.py", line 146, in Run
    creds = auth_util.DoInstalledAppBrowserFlow(launch_browser, scopes)
  File "/tmp/google-cloud-sdk/lib/googlecloudsdk/api_lib/auth/util.py", line 95, in DoInstalledAppBrowserFlow
    client_secret=client_secret)
  File "/tmp/google-cloud-sdk/lib/googlecloudsdk/core/credentials/store.py", line 526, in AcquireFromWebFlow
    return RunWebFlow(webflow, launch_browser=launch_browser)
  File "/tmp/google-cloud-sdk/lib/googlecloudsdk/core/credentials/store.py", line 549, in RunWebFlow
    raise FlowError(e)
FlowError: invalid_requestMissing required parameter: code
2018-08-16 21:51:07,187 ERROR    root            (gcloud.auth.login) invalid_requestMissing required parameter: code
2018-08-16 21:51:07,446 DEBUG    root            Metrics reporting process started...
